assets/src/components/activities/types.ts

```diff
@@ -1,17 +1,28 @@
 import { create, ID, Identifiable, ModelElement, Selection } from 'data/content/model';
 import { ResourceContext } from 'data/content/resource';
 import { ResourceId } from 'data/types';
-import { InsertOperation } from 'utils/undo';
+import { UndoOperation } from 'utils/undo';
 import guid from 'utils/guid';
 
 export type PostUndoable = (undoable: Undoable) => void;
 
 export type Undoable = {
   type: 'Undoable';
   description: string;
-  operations: InsertOperation[];
+  operations: UndoOperation[];
 }
 
+export function makeUndoable(description: string, operations: UndoOperation[]) : Undoable {
+  return {
+    type: 'Undoable',
+    description,
+    operations
+  };
+}
+
+export function clone(o : any) {
```

Comment:
I often find myself doing this, should we put it in utils/common or somewhere similar instead?

Reply:
Yes, agreed.